### PR TITLE
Edit UserGuide

### PR DESCRIPTION
**Pull request by GSgiansen**
Edited UserGuide to reference previously decided User Guide

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -2,196 +2,135 @@
 layout: page
 title: User Guide
 ---
-
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
-
+lesSOn is a **flashcard software aimed to help individuals with their memory work in school, optimized for use via a Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI). If you can type fast, lesSon can get your contact management tasks done faster than traditional GUI apps.
```
Comment (Dioclei) [inline docs/UserGuide.md:5]: Can we standardise the spelling of lesSON?

**Review by Dioclei — COMMENTED** [review]
Looks good to me other than the spelling of lesSON.

**Review by Dioclei — CHANGES_REQUESTED** [review]
Can we standardise the spelling of lesSON?

File: docs/UserGuide.md (hunk 1 above)
Comment (GSgiansen) [reply docs/UserGuide.md]: Sure! my bad overlooking this

**Review by Dioclei — APPROVED** [review]
Looks good to me!

**Pull request merged**